ui/DesignSystem/Component/Transaction/Accordion.svelte

```diff
@@ -0,0 +1,177 @@
+  export let transactions = null;
+  export let style = null;
+
+  const stateToColor = {
+    pending: "var(--color-caution)",
+    success: "var(--color-positive)",
+    error: "var(--color-negative)"
+  };
+
+  const stateToDescription = {
+    pending: "Pending",
+    success: "Success",
+    error: "Error"
+  };
+
+  const stateToIconState = {
+    pending: "caution",
+    success: "positive",
+    error: "negative"
+  };
+
+  const stateToSummary = {
+    pending: "pending",
+    success: "succeeded",
+    error: "failed"
+  };
+
+  const txSummary = {
+    pending: { count: 0, progress: 0 },
+    success: { count: 0, progress: 0 },
+    error: { count: 0, progress: 0 }
+  };
+
+  const fillSummary = () => {
+    transactions.forEach(tx => {
+      txSummary[tx.state].count += 1;
+      txSummary[tx.state].progress += tx.progress || 0;
+    });
+    txSummary["success"].progress = txSummary["success"].count * 100;
+  };
+
+  fillSummary();
+
+  const progressSummary =
+    txSummary["error"].count === transactions.length
+      ? null
+      : (txSummary["pending"].progress + txSummary["success"].progress) /
+        transactions.length;
+
+  const iconState =
+    txSummary["error"].count > 0
+      ? "negative"
+      : txSummary["pending"].count > 0
+      ? "caution"
+      : "positive";
+
+  const summaryState =
+    txSummary["pending"].count > 0
+      ? "pending"
+      : txSummary["error"].count > 0
+      ? "error"
+      : "success";
+
+  const summaryText = () => {
+    if (txSummary[summaryState].count > 1) {
+      return `${txSummary[summaryState].count} Transactions ${stateToSummary[summaryState]}`;
+    } else {
+      return `Transaction ${stateToSummary[summaryState]}`;
+    }
+  };
+
+  let hidden = true;
+
+  const hideCards = () => {
+    hidden = hidden ? false : true;
+  };
+</script>
+
+<style>
+  .pipeline {
+    border: 1px solid var(--color-foreground-level-3);
+    border-radius: 4px;
+    box-shadow: 0px 4px 8px var(--color-foreground-level-3-opacity-08);
+    width: 274px;
+    cursor: pointer;
+  }
+
+  .negative {
+    border: 1px solid var(--color-negative);
+  }
+
+  .cards {
+    background-color: var(--color-foreground-level-1);
+    border-radius: 3px 3px 0 0;
+  }
+
+  .card {
+    border-bottom: 1px solid var(--color-foreground-level-3);
+    height: 64px;
+  }
+
+  .summary {
+    height: 56px;
+    background-color: var(--color-background);
+    border-radius: 3px;
+  }
+
+  .description {
+    align-self: center;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .hidden {
+    display: none;
+  }
+</style>
+
+<div class="pipeline" class:negative={iconState === 'negative'} {style}>
+  <div class="cards" class:hidden>
+    {#each transactions as tx}
+      <div
+        class="card"
+        on:click={() => {
+          push(path.transactions(tx.id));
+        }}>
+        <Flex>
+          <div slot="left">
+            <Flex align="left">
```

Comment:
You could avoid wrapping the content within another `<Flex align="left">
 ` by setting the parent `div`'s inline style to: `style="display: flex"`.